doc/nrf/protocols/thread/device_types.rst

```diff
@@ -8,15 +8,19 @@ Thread device types
    :depth: 2
 
 Thread devices can be either Full Thread Devices (FTD) or Minimal Thread Devices (MTD).
-The devices are divided based on their ability to exist in a network without a parent, determined by whether the device maintains a routing table (doesn't need a parent) or not (needs a parent).
-FTDs are the devices that maintain a routing table, while MTDs do not, forwarding all their messages to a parent instead.
+The devices are divided based on the following criteria:
 
-Another type of Thread device classification is based on the device's ability to route messages:
+* How they exist in a Thread network - whether the device keeps its radio enabled all the time.
+* How they route messages - whether it maintains a routing table (can route messages) or not (does not route messages).
+
+FTDs are the devices that keep their radios enabled all the time and maintain a routing table, while MTDs do not have their radio enabled at all times and they do not maintain a routing table, consequently forwarding all their messages to a parent instead.
```

Comment:
```suggestion
FTDs are the devices that keep their radios enabled all the time and maintain a routing table, while MTDs may have their radio enabled at all times and they do not maintain a routing table, consequently forwarding all their messages to a parent instead.
```

MED is one of MTD role which keeps its receiver on all the time - so "may" is more propriate.